src/components/dialogs/parameters/sensi/sensitivity-table.tsx

```diff
@@ -68,57 +65,40 @@ const SensitivityTable: FunctionComponent<SensitivityTableProps> = ({
                 border: 'solid 0px rgba(0,0,0,0.1)',
             }}
         >
-            <Table stickyHeader size="small">
+            <Table stickyHeader size="small" sx={{ tableLayout: 'fixed' }}>
                 <TableHead>
-                    <TableRow>
-                        {columnsDefinition.map((column: any) => (
-                            <TableCell>
-                                <Box
-                                    sx={{
-                                        backgroundColor: column.color,
-                                    }}
-                                >
-                                    <FormattedMessage id={column.label} />
-                                </Box>
-                            </TableCell>
-                        ))}
-                        <TableCell>
-                            <Tooltip
-                                title={intl.formatMessage({
-                                    id: 'AddRows',
-                                })}
-                            >
-                                <Box>
-                                    <IconButton
-                                        color="primary"
-                                        onClick={handleAddRowsButton}
-                                    >
-                                        <AddCircleIcon
-                                            sx={{
-                                                color:
-                                                    getLocalStorageTheme() ===
-                                                    DARK_THEME
-                                                        ? 'white'
-                                                        : 'black',
-                                            }}
-                                        />
-                                    </IconButton>
-                                </Box>
-                            </Tooltip>
+                    {columnsDefinition.map((column: IColumnsDef) => (
+                        <TableCell
+                            key={column.dataKey}
+                            sx={{ width: column.width, textAlign: 'center' }}
+                        >
+                            <Box>{column.label}</Box>
                         </TableCell>
-                    </TableRow>
+                    ))}
+                    <TableCell sx={{ width: '5rem', textAlign: 'center' }}>
+                        <Tooltip
+                            title={intl.formatMessage({
+                                id: 'AddRows',
+                            })}
+                        >
+                            <IconButton onClick={handleAddRowsButton}>
+                                <AddCircleIcon />
```

Comment:
just curious: why did this button become black & white, and not blue like in other forms, with the default theme ?

Reply:
requested by PO